packages/fast-usdc/test/exos/status-manager.test.ts

```diff
@@ -10,7 +10,7 @@ import type { CctpTxEvidence } from '../../src/types.js';
 
 type Common = Awaited<ReturnType<typeof commonSetup>>;
 type TestContext = {
-  statusNode: ERef<StorageNode>;
+  transactionsNode: ERef<StorageNode>;
```

Comment:
odd that we spell it out internally but not externally.

Reply:
`txs` is embedded in a string that's already very long and distinct so the motivation was there.

I agree it's probably better to make them match. I'd rather make this `txsNode` than have `published.fastUsdc.transactions.0x19330d10D9Cc8751218eaf51E8885D058642E08A`